test/system/272-system-connection.bats

```diff
@@ -232,9 +232,12 @@ $c2[ ]\+tcp://localhost:54321[ ]\+true" \
     run_podman system connection rm env-override
     run_podman system connection rm cli-override
 
-    _run_podman_remote 125 --remote ps
-    assert "$output" =~ "/run/[a-z0-9/]*podman/podman.sock"\
-        "test absence of default connection"
+    # With no default system connection. Only applicable on podman, not -remote
+    if ! is_remote; then
+        _run_podman_remote 125 --remote ps
+        assert "$output" =~ "/run/[a-z0-9/]*podman/podman.sock"\
+               "test absence of default connection"
+    fi
```

Comment:
This is not the issue. In fact all test here are already skipped on the local case.

The problem is that gating uses default /run/user/$UID/podman/podman.sock while upstream CI uses a socket under /tmp and thus in upstream CI this command fails while in gating it works.
I guess the fix here is to run `--format {{.Host.RemoteSocket.Path}}`, if the command exits 0 than just compare the path and if it errors we should have the same path in the error like we do match right now.